components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/PasswordPolicyConstants.java

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
We need to use the new Licenses header in all newly added files.

Reply:
This header complies with the header format in mail "Re: License Header for WSO2 code"